index.js

```diff
@@ -1,8 +1,11 @@
 module.exports = `
+
 scalar Date
 
 scalar JSON
 
+directive @deprecated(reason: String) on INPUT_FIELD_DEFINITION | ARGUMENT_DEFINITION |ENUM_VALUE | FIELD_DEFINITION
```

Comment:
Is the `(reason: String)` argument needed for it to work?

Reply:
Yep, it failed the yarn test if I removed it with a "Unknown argument "reason" on directive "@deprecated"." error everytime reason was then used

Reply:
Ah cool that makes sense. 👍